jinahub/encoders/text/TransformerTorchEncoder/transform_encoder.py

```diff
@@ -153,6 +172,12 @@ def _generate_input_tokens(self, texts):
             truncation=True,
             return_tensors='pt',
         )
+
+        if self.is_distill_bert:
+            # DistilBERT doesn’t have token_type_ids, you don’t need to indicate which token belongs to which segment.
+            # https://huggingface.co/transformers/model_doc/distilbert.html
+            del input_tokens['token_type_ids']
```

Comment:
As I say above, we should move this to a separate PR, but since it's here: why is this needed? What breaks without it?